declarations/WebpackOptions.d.ts

```diff
@@ -44,9 +44,14 @@ export type DevServer =
 			[k: string]: any;
 	  };
 /**
- * A developer tool to enhance debugging (false | eval | [inline-|hidden-|eval-][nosources-][cheap-[module-]]source-map).
+ * A developer tool to enhance debugging (false | eval | [inline-|hidden-|eval-][nosources-][cheap-[module-]]source-map | Object).
  */
-export type DevTool = (false | "eval") | string;
+export type DevTool =
+	| (false | "eval")
+	| string
+	| (import("../lib/SourceMapDevToolPlugin").SourceMapDevToolPluginOptions & {
+			eval?: boolean;
```

Comment:
if eval in object literal is not defined, we can fallback to `eval: false`

Reply:
I see that we use source-map-dev-tool-plugin for this instead when eval, im good with that! sorry for spam.